server/repository/src/db_diesel/program_indicator_row.rs

```diff
@@ -0,0 +1,69 @@
+use super::StorageConnection;
+
+use crate::{repository_error::RepositoryError, Upsert};
+
+use diesel::prelude::*;
+
+table! {
+    program_indicator (id) {
+        id -> Text,
+        program_id -> Text,
+        code -> Text,
```

Comment:
In the migration you have code as Nullable

Reply:
TRUE